istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -434,12 +397,7 @@ func allConfigCmd(ctx cli.Context) *cobra.Command {
 					if err != nil {
 						return err
 					}
-					ztunnelPod := ambientutil.IsZtunnelPod(kubeClient, podName, podNamespace)
```

Comment:
Do we need to keep an info message here to tell users to use `zc`? Some people may have already used this to obtain the ztunnel dump, but now it doesn't work anymore.

Reply:
IMO making an API call to just add a warning here is a bit excessive; ambient is going beta with `ztunnel-config` so there isn't much usage currently anyways